packages/react-storage/src/styles/storage-browser.css

```
@@ -119,12 +119,18 @@
   margin: 0;
 }
 
-.storage-browser__refresh {
+.storage-browser__locations-view__data-refresh,
+.storage-browser__locations-detail-view__data-refresh {
```

Nit: I think we will have to re-evaluate on a whole at some point anyway so this isn't super important to get right just yet
```suggestion
.storage-browser__locations-view-data-refresh,
.storage-browser__locations-detail-view-data-refresh {
```

---

sounds good, updated